crates/optimism/consensus/src/lib.rs

```diff
@@ -96,7 +99,15 @@ impl Consensus for OptimismBeaconConsensus {
         Ok(())
     }
 
-    fn validate_block(&self, block: &SealedBlock) -> Result<(), ConsensusError> {
-        validation::validate_block_standalone(block, &self.chain_spec)
+    fn validate_block_pre_execution(&self, block: &SealedBlock) -> Result<(), ConsensusError> {
+        validation::validate_block_pre_execution(block, &self.chain_spec)
+    }
+
+    fn validate_block_post_execution(
+        &self,
+        block: &BlockWithSenders,
+        receipts: &[Receipt],
+    ) -> Result<(), ConsensusError> {
+        validation::validate_block_post_execution(block, &self.chain_spec, receipts)
```

Comment:
this is incorrect, the receipts root is different for OP

Reply:
i see, fixed